docs/get-started/getting-started-workload-qualification.md

```diff
@@ -0,0 +1,135 @@
+But it is very convenient because you do not need a CPU or GPU Spark cluster to run them.
+
+## 2. Function `explainPotentialGPUPlan` 
+
+### Requirement
+
+- A Spark 3.x CPU cluster
+- A pair of `rapids-4-spark` and `cudf` jars
+
+### How to use
+
+In Spark RAPIDS accelerator 21.12, a new function named `explainPotentialGPUPlan` is added which can help us understand 
+the potential GPU plan and if there are any not-supported features on a CPU cluster.
+Basically it can print the logs which are the same as the driver logs with `spark.rapids.sql.explain=all`.
+
+1. In `spark-shell`, add the `rapids-4-spark` and `cudf` jars into --jars option or put them in Spark related classpath.
+
+   For example:
+
+   ```bash
+   spark-shell --jars /PathTo/cudf-<version>.jar,/PathTo/rapids-4-spark_<version>.jar
+   ```
+
+2. Test if the class can be successfully loaded or not.
+
+   ```scala
+   import com.nvidia.spark.rapids.ExplainPlan.explainPotentialGPUPlan
+   ```
+
+3. Enable Spark RAPIDS related parameters to allow supporting existing features.
+   
+   For example, if your jobs have `double`/`float`/`decimal` operators together with some scala UDFs, you can set 
+   below parameters:
+   
+   ```scala
+   spark.conf.set("spark.rapids.sql.incompatibleOps.enabled", true)
+   spark.conf.set("spark.rapids.sql.variableFloatAgg.enabled", true)
+   spark.conf.set("spark.rapids.sql.decimalType.enabled", true)
+   spark.conf.set("spark.rapids.sql.castFloatToDecimal.enabled",true)
+   spark.conf.set("spark.rapids.sql.castDecimalToFloat.enabled",true)
+   spark.conf.set("spark.rapids.sql.udfCompiler.enabled",true)
+   ```
+   
+   Please refer to [config doc](../configs.md) for details of those Spark RAPIDS related parameters.
+
+4. Run the function `explainPotentialGPUPlan` on top of Dataframe.
+
+   For example:
+
+   ```scala
+   scala> val df_multi=spark.sql("SELECT value82*value63 FROM df2 union SELECT value82+value63 FROM df2")
+   df_multi: org.apache.spark.sql.DataFrame = [(CAST(value82 AS DECIMAL(9,3)) * CAST(value63 AS DECIMAL(9,3))): decimal(15,5)]
+
+   scala> val output=com.nvidia.spark.rapids.ExplainPlan.explainPotentialGPUPlan(df_multi)
+   scala> println(output)
+   ```
+
+   Below are sample driver log messages starting with `!` which indicate the unsupported features in this version:
+   
+   ```
+   !Exec <ProjectExec> cannot run on GPU because not all expressions can be replaced
+     !Expression <Multiply> (promote_precision(cast(value82#30 as decimal(9,3))) * promote_precision(cast(value63#31 as decimal(9,3)))) cannot run on GPU because The actual output precision of the multiply ilarge to fit on the GPU DecimalType(19,6)
+   ```
+
+This log can show you which operators(on what data type) can not run on GPU and what is the reason.
+If it shows a specific Spark RAPIDS parameter which can be turned on to enable that feature, you can enable that parameter 
+and try the tool again.
```

Comment:
Probably want to mention here that one shouldn't blindly enable parameters that are disabled by default without understanding the risks and applicability of that parameters ramifications to their specific application.

Reply:
I added some words to ask users to understand the risks of the parameters. How is now?

Reply:
This is better.  Would be nice to link to the configs.md document here so users can know where to lookup any parameters they may find in the output.

Reply:
Added the configs doc link here.